tensorflow/java/src/main/java/org/tensorflow/SavedModelBundle.java

```diff
@@ -0,0 +1,94 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+package org.tensorflow;
+
+import java.util.Set;
+
+/**
+ * SavedModel representation once the SavedModel is loaded from storage.
```

Comment:
Let's add a little more descriptive detail about SavedModels. Perhaps something like (with a link to MetaGraphDef):

/**
 * SavedModel represents a model loaded from storage.
 *
 * The model consists of a description of the computation (a {@link Graph}) , a {@link Session}
 * with tensors (e.g., parameters or variables in the graph) initialized to values saved on storage
 * and a description (<a href="https://www.tensorflow.org/code/tensorflow/core/protobuf/meta_graph.proto">MetaGraphDef</a>) of the model.
 */